docker/wiremock/stubs/mappings/retreiveNoneExistingPatientDocument.json

```diff
@@ -11,10 +11,9 @@
       "Server":"nginx",
       "Date":"Tue, 12 Jan 2021 11:36:57 GMT",
       "Content-Type":"application/fhir+json;charset=UTF-8",
-      "Content-Length":"291",
+      "Content-Length":"528",
```

Comment:
You normally would not hardcode a content-length unless you're trying to test an invalid response header. Surely wiremock calculates this by default?

Reply:
I think it will be the best to also remove the content length. It is not necessary for a response body to be returned.